app/components/documents/document-preview/document-preview-modal.hbs

```diff
@@ -21,7 +21,7 @@
   <:body>
     <div class="auk-panel-layout">
       <div class="auk-panel-layout__main-content">
-        <Documents::DocumentView @file={{await this.selectedVersion.file}} />
```

Comment:
Waarom deze wijziging @sergiofenoll ?

Reply:
Met `await` → bij veranderen van file werd de `DocumentView` niet opnieuw gerendered, dus de gebruiker ziet de nieuwe file niet tot een refresh.
Zonder `await` → automatisch re-render (wrss omdat `this.selectedVersion.file` veranderd is, dus de autotracking kicks in?) en geen refresh nodig